docs/cloud/dev/spryker-cloud-commerce-os/monitoring-issues-and-informing-about-alerts.md

```diff
@@ -1,107 +1,107 @@
 ---
 title: Monitoring issues and informing about alerts
-description: Learn how issues with environments are monitored and how customers are informed about alerts on PaaS
+DESCRIPTION: Learn how issues with environments are monitored and how customers are informed about alerts on PaaS
 template: concept-topic-template
 ---
 
-Every Spryker cloud environment is monitored by the monitoring systems and a dedicated 24/7 team that ensures that environments run stable for our customers. In case of problems, this team can take action autonomously or inform customers and partners if there is a need for further action on their side. This article explains in more detail how monitoring and alerting works on Spryker PaaS.
+Every Spryker cloud environment is monitored by the monitoring systems and a dedicated 24/7 team that ensures that environments run stable. In case of problems, this team takes action autonomously or inform you if there is a need for further action on their side. This article explains in details how monitoring and alerting works on Spryker PaaS.
```

Comment:
```suggestion
Every Spryker cloud environment is monitored by the monitoring systems and a dedicated 24/7 team that ensures that environments run stable. In case of problems, this team takes action autonomously or inform you if there is a need for further action on their side. This document explains in details how monitoring and alerting works on Spryker PaaS.
```